src/client/feed/Feed.js

```diff
@@ -65,6 +96,7 @@ class Tweet extends Component {
 const mapStateToProps = (state) => {
   return {
     feed: state.feed,
+    hiddenWords: state.mostUsedWords.wordsToHide,

```

Comment:
I personally think that mostUsedWords should be part of the feed reducer, thoughts @Winwardo ?

ie state.feed.mostUsedWords